apps/network-alerts/js/index.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
+// See reference TS in following document for GSM and CDMA:
+// [1] GSM: http://www.etsi.org/deliver/etsi_ts/123000_123099/123041/
+//     11.06.00_60/ts_123041v110600p.pdf, chapter 9.4.1.2.2 Message Identifier
+//       
+// [2] CDMA: http://www.3gpp2.org/public_html/specs/
+//     C.R1001-G_v1.0_Param_Administration.pdf, chapter 9.3.3 Commercial Mobile
+
 (function() {
   const CMAS_ID = 'emergency-alert-title',
         CMAS_ENABLED_KEY = 'cmas.enabled';
 
-  // See reference TS in following document, table under section 9.4.1.2.2.
-  // http://www.etsi.org/deliver/etsi_ts/123000_123099/123041/11.06.00_60/ts_123041v110600p.pdf
   const GSM_PRESIDENTIAL_ALERTS = [4370, 4383];
+  const CDMA_PRESIDENTIAL_ALERTS = [4096];
```

Comment:
because CDMA is defined using hexadecimal values, maybe you should use it instead? `0x1000` instead of `4096` looks good :)